### ISSUE TITLE
Gamification #feature-request

### ISSUE THREAD

**Issue by YohananDiamond** [iSoron/uhabits#450]
I'd like to have a XP & Levels system, I think it's more motivating :)
Also, if, for example, the user didn't do the habit on a certain day (selected by the user, like on some weekdays etc.), the player could lose XP (but never go down on level)

**Comment by goldmanm** [comment]
One simpler way to gamify it would be to just show the % completion as a number inside or over the loop. This is easier to quantify than the ring, and users can think of this as experience for various skills without complicating the inner workings at all.

**Comment by malnvenshorn** [comment]
I second the idea for some kind of gamification. I found myself getting sloppy on my habits mainly because there is no real award or penalty involved and I don't feel attached to a plain number or percentage graph.

I don't think that the implementation has to be a clone of Habitica with XP, levels, skills, etc. Instead I could think of taking care of a plant or pet. E.g. earning coins when checking a habit. With that coins you have to buy food, water, fertilizer or what so ever to take care of your pet/plant or otherwise it would die over time.

**Comment by ImAnnoying2** [comment]
While a good idea overall,  I think this app's primary strength is its simplicity. What you're proposing can be easily accomplished by adding a Daily to Habitica called "Done all my Loop habits."

**Comment by malnvenshorn** [comment]
The issue with Habitica is it's a pure privacy nightmare.

**Comment by Phieldar** [comment]
I agree with ImAnnoying2. This app strenght is its simplicity. In fact, I prefer this one to Habitica because it is not full of xp, gold and stuff like that. There are plenty of RPG style habit tracker (Do It now, HabitRPG, etc). I prefer this app, it is focused, simple and with no fluff.
